triage/2021-07-20.md

```diff
@@ -0,0 +1,73 @@
+- It seems like these results were [expected](https://github.com/rust-lang/rust/pull/87044#issuecomment-879407381), and they make sense given the nature of the PR.
+
+Update Rust Float-Parsing Algorithms to use the Eisel-Lemire algorithm. [#86761](https://github.com/rust-lang/rust/issues/86761)
+- Moderate regression in [instruction counts](https://perf.rust-lang.org/compare.html?start=64d171b8a419eb6cb872ab579398eff8a741bbc6&end=f502bd3abd12111bbfae0974db018c165a977c0e&stat=instructions:u) (up to 3.2% on `incr-patched: b9b3e592dd cherry picked` builds of `style-servo-debug`)
+- Moderate improvement in [instruction counts](https://perf.rust-lang.org/compare.html?start=64d171b8a419eb6cb872ab579398eff8a741bbc6&end=f502bd3abd12111bbfae0974db018c165a977c0e&stat=instructions:u) (up to -1.4% on `full` builds of `piston-image-opt`)
+- A perf run was done on the orignal PR, but the run there did not predict the regression that was now observed to `style-servo-debug` when this PR landed on nightly.
+- Left a comment, but I am guessing that `style-servo-debug` may need a bigger noise threshold (or rather, sensitivity to perturbations) than what we currently use.
+
+Move OnDiskCache to rustc_query_impl. [#86698](https://github.com/rust-lang/rust/issues/86698)
+- Moderate improvement in [instruction counts](https://perf.rust-lang.org/compare.html?start=5a8a44196b3cf099f8c9b0156bd902eaec0b4e5f&end=18073052d8c3544ccb73effd289ed3acda0d66c0&stat=instructions:u) (up to -1.4% on `incr-unchanged` builds of `ctfe-stress-4-check`)
+- Moderate regression in [instruction counts](https://perf.rust-lang.org/compare.html?start=5a8a44196b3cf099f8c9b0156bd902eaec0b4e5f&end=18073052d8c3544ccb73effd289ed3acda0d66c0&stat=instructions:u) (up to 1.1% on `incr-unchanged` builds of `deeply-nested-async-opt`)
+- A perf run was done on the original PR that predicted a much better outcome here.
+- Left a comment asking PR author if they have any idea about the discrepancy.
+
+#### Untriaged Pull Requests
+
+- [#87153 \[debuginfo\] Emit associated type bindings in trait object type names.](https://github.com/rust-lang/rust/pull/87153)
+- [#86993 Replace associated item bound vars with placeholders when projecting](https://github.com/rust-lang/rust/pull/86993)
+- [#86777 Include terminators in instance size estimate](https://github.com/rust-lang/rust/pull/86777)
+- [#86588 Rollup of 8 pull requests](https://github.com/rust-lang/rust/pull/86588)
+- [#86034 Change entry point to 🛡️ against 💥 💥-payloads](https://github.com/rust-lang/rust/pull/86034)
+- [#84560 Inline Iterator as IntoIterator.](https://github.com/rust-lang/rust/pull/84560)
+
+#### Nags requiring follow up
+
+* [PR #87244 followup](https://github.com/rust-lang/rust/pull/87244#issuecomment-883635813)
+* [PR #86698 followup](https://github.com/rust-lang/rust/pull/86698#issuecomment-883647259)
```

Comment:
I added `perf-regression` labels to these meaning that if they aren't addressed they'll show up in next week's untriaged pull request's section - which means they don't need to be here.

Reply:
okay, removed the nags